sdk/react-native/README.md

```diff
@@ -223,32 +254,43 @@ this._picovoice.delete();
 
 ## Custom Model Integration
 
-To add custom models to your React Native application you'll need to add the files to your platform projects. Android models must be added to `./android/app/src/main/res/raw/`, while iOS models can be added anywhere under `./ios`, but must be included as a bundled resource in your iOS project. Then in your app javascript code, using the [react-native-fs](https://www.npmjs.com/package/react-native-fs) package, retreive the files like so:
+To add a custom models to your React Native application you'll need to add the rhn files to your platform projects.
+
+### Adding Android Models
+
+Android custom models must be added to [`./android/app/src/main/assets/`](android/app/src/main/assets/).
+
+### Adding iOS Models
+
+iOS models can be added anywhere under [`./ios`](ios), but it must be included as a bundled resource. 
+The easiest way to include a bundled resource in the iOS project is to:
+
+1. Open XCode.
+2. Either:
+  - Drag and Drop the model/keyword file to the navigation tab.
+  - Right click on the navigation tab, and click `Add Files To ...`.
+
+This will bundle your models together when the app is built.
+
+### Using Custom Models
+
 ```javascript
-const RNFS = require('react-native-fs');
 
-let wakeWordName = 'keyword';
-let wakeWordFilename = wakeWordName;
+const accessKey = "${ACCESS_KEY}"; // obtained from Picovoice Console (https://picovoice.ai/console/)
+
 let wakeWordPath = '';
-let contextName = 'context';
-let contextFilename = contextName;
 let contextPath = '';
 
 if (Platform.OS == 'android') {
     // for Android, extract resources from APK
-    wakeWordFilename += '_android.ppn';
-    wakeWordPath = `${RNFS.DocumentDirectoryPath}/${wakeWordFilename}`;
-    await RNFS.copyFileRes(wakeWordFilename, wakeWordPath);
+    wakeWordPath = `path_to_keyword_android.ppn`;
```

Comment:
you may just want to say `keyword_android.ppn` instead to make it clear that they don't have to give the path to the assets or ios folder.